spec/components/post_creator_spec.rb

```diff
@@ -776,6 +776,22 @@
 
       expect(post.topic.topic_allowed_users.where(user_id: admin2.id).count).to eq(0)
     end
+
+    it 'does not increase posts count for small actions' do
```

Comment:
Can you also test for the number of small actions in that PM? This would make sure we are actually skipping them.

Reply:
Did that and also added a few more posts to ensure they are counted.